test/parallel/test-http-write-head.js

```diff
@@ -18,6 +18,17 @@ var s = http.createServer(function(req, res) {
   }
   assert.ok(threw, 'Non-string names should throw');
 
+  // undefined value should throw, via 979d0ca8
+  threw = false;
+  try {
```

Comment:
as it is the same style as the above section, @cjihrig would you object to me merging this in as-is?